examples/src/features/module_nesting.rs

```diff
@@ -2,20 +2,20 @@ use casper_event_standard::Event;
 use odra::casper_event_standard;
```

Comment:
I think this line might be avoided, if `casper_event_standard` is exported via `prelude`.